docs/source/quick-start/opf.rst

```diff
@@ -34,7 +34,7 @@ using the :class:`.ModelFactory`.
 
 .. literalinclude:: ../../examples/opf/create-model-example.py
 
-The resulting ``model`` will be an instance of :class:`.CLAModel`.
+The resulting ``model`` will be an instance of :class:`.SPTMModel`.
```

Comment:
@subutai @mrcslws @scottpurdy I would like it if you guys could talk in the office and come up with a name you agree on. Once you have one, I will update the PR.